pkgs/async/pubspec.yaml

```diff
@@ -0,0 +1,20 @@
+name: async
+version: 2.12.0
+description: Utility functions and classes related to the 'dart:async' library.
+repository: https://github.com/dart-lang/core/main/pkgs/async
+
+environment:
+  sdk: ^3.4.0
+
+dependencies:
+  collection: ^1.15.0
+  meta: ^1.3.0
+
+dev_dependencies:
+  dart_flutter_team_lints: ^3.0.0
+  fake_async: ^1.2.0
+  stack_trace: ^1.10.0
+  test: ^1.16.6
+
+topics:
```

Comment:
As a nit, perhaps move this up above the `dependencies` section? That's what many of our other packages do.